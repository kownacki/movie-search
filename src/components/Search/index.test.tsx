import React from 'react';
import {shallow} from 'enzyme';
import {Search} from './index';

describe('Search', () => {
  it('renders correctly', () => {
    const searchForMoviesStub = () => {'nothing'};
    const search = shallow(<Search searchForMovies={searchForMoviesStub}/>);
    expect(search).toMatchSnapshot();
  });
});
