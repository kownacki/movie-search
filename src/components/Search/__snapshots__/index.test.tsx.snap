// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search renders correctly 1`] = `
<div>
  <styled.input
    onChange={[Function]}
    onKeyDown={[Function]}
    placeholder="Enter movie title"
    type="text"
  />
  <styled.button
    disabled={true}
    onClick={[Function]}
  >
    Search
  </styled.button>
  <Connect(SearchResults) />
</div>
`;
